pkg/cmd/cli/cmd/rollback.go

```diff
@@ -0,0 +1,85 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kubecmd "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd"
+
+	latestapi "github.com/openshift/origin/pkg/api/latest"
+	client "github.com/openshift/origin/pkg/cmd/client"
+	deployapi "github.com/openshift/origin/pkg/deploy/api"
+)
+
+const rollbackLongDesc = `
+Rolls back a deploymentConfig to match a previous deployment. By default, only the pod
+template of the specified deployment will be included in the rollback.
+
+Examples:
+  $ %[1]s %[2]s deployment-2
+  ...
+
+  $ %[1]s %[2]s deployment-3 --include-strategy=true
+  ...
+
+  $ %[1]s %[2]s deployment-1 --output=json
+  ...
+`
+
+func NewCmdRollback(parentName string, name string, f *Factory, out io.Writer) *cobra.Command {
+	rollback := &deployapi.DeploymentConfigRollback{}
+
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("%s <from-deployment>", name),
+		Short: "Rolls back a deploymentConfig to match a previous deployment.",
+		Long:  fmt.Sprintf(rollbackLongDesc, parentName, name),
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) == 0 || len(args[0]) == 0 {
+				usageError(cmd, "A deployment name is required.")
+			}
+
+			rollback.Spec.From.Name = args[0]
+
+			output := kubecmd.GetFlagString(cmd, "output")
+
+			osClient, _, err := f.Clients(cmd)
+			checkErr(err)
+
+			namespace := getOriginNamespace(cmd)
+
+			newConfig, err := osClient.DeploymentConfigs(namespace).Rollback(rollback)
+			checkErr(err)
+
+			// If an output format is specified, display the rollback config JSON and exit
+			// WITHOUT performing a rollback.
+			if len(output) > 0 {
+				switch output {
+				case "json":
+					json, encodeErr := latestapi.Codec.Encode(newConfig)
+					checkErr(encodeErr)
+
+					printer := client.JSONPrinter{}
+					printErr := printer.Print(json, out)
+					checkErr(printErr)
+				default:
+					usageError(cmd, "supported output types: json")
+				}
+
+				return
+			}
+
+			_, updateErr := osClient.DeploymentConfigs(namespace).Update(newConfig)
+			checkErr(updateErr)
+		},
+	}
+
+	cmd.Flags().BoolVar(&rollback.Spec.IncludeTemplate, "include-template", true, "Include the previous deployment's pod template in the rollback")
```

Comment:
Removed the option, made it default to true. I also greatly expanded the help information.